formats/opentype.fathom

```diff
@@ -418,9 +418,8 @@ let cmap_subtable_format4 = fun (platform : Repr platform_id) => {
     id_delta <- array16 seg_count s16be,
     /// Offsets into glyphIdArray or 0
     id_range_offsets <- array16 seg_count u16be,
-    // TODO: Needs length limiting formats
-    // /// Glyph index array (arbitrary length)
-    // glyph_id_array[ ]  <- u16be,
+    /// Glyph index array (arbitrary length)
+    glyph_id_array <- repeat_until_end u16be,
```

Comment:
I'm guessing without #310 this just goes until the end of the binary data?

Reply:
Oh right… I was thinking that it stopped at the end of the table… perhaps this change was premature.